app/views/opportunities/_questions.erb

```diff
@@ -10,9 +10,14 @@
 <% end %>
 
 <% if policy(@opportunity).ask_question? %>
-  <div class='opportunity_section'>
+  <div class='opportunity_section' id="ask_question">
```

Comment:
But we're styling it: https://github.com/dobtco/dispatch/blob/ecb1e6d7717eed4859f55aa34875a549b8fe8b6f/themes/dvl-core/assets/stylesheets/theme/components/questions.scss#L8